editor/src/clj/editor/gl/vertex2.clj

```diff
@@ -83,7 +82,7 @@
   (position! [this position])
   (version [this]))
 
-(deftype VertexBuffer [vertex-description usage ^ByteBuffer buf ^{:unsynchronized-mutable true} version]
+(deftype VertexBuffer [vertex-description usage ^Buffer buf ^{:unsynchronized-mutable true} version]
```

Comment:
The OpenGL methods require this to be a `Buffer`. If we relax the requirement here we can use a `FloatBuffer` as well as a `ByteBuffer`.